bolt-servlet/src/test/java/samples/OAuthCallbacksSample.java

```diff
@@ -0,0 +1,89 @@
+package samples;
+
+import com.slack.api.SlackConfig;
+import com.slack.api.bolt.App;
+import com.slack.api.bolt.AppConfig;
+import com.slack.api.bolt.response.Response;
+import com.slack.api.bolt.service.builtin.oauth.OAuthV2AccessErrorHandler;
+import com.slack.api.model.event.AppMentionEvent;
+import com.slack.api.util.json.GsonFactory;
+import lombok.extern.slf4j.Slf4j;
+import util.TestSlackAppServer;
+
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+@Slf4j
+public class OAuthCallbacksSample {
+
+    public static void main(String[] args) throws Exception {
+        String jsonString = Files.readAllLines(Paths.get("bolt-servlet/src/test/resources/appConfig_oauth.json"))
```

Comment:
I cannot seem to find this file... where can I find it?

Reply:
@filmaj This is a working example with the Slack production server. You can place the file with your own real credentials.

Reply:
FYI: Those files exist in my local repo but the JSON files are excluded by .gitignore rules